addons/website_blog/models/website_snippet_filter.py

```diff
@@ -55,3 +55,10 @@ def _get_hardcoded_sample(self, model):
                 # merge definitions
             samples = merged
         return samples
+
+    @api.model
+    def default_get(self, fields):
+        defaults = super().default_get(fields)
+        if 'field_names' in defaults and self._context.get('model') == 'blog.post':
+            defaults['field_names'] = 'name,teaser,subtitle'
```

Comment:
There are missing spaces after the commas. (here and in the others `default_get` function in other apps)